roiextractors/extractors/nwbextractors/nwbextractors.py

```diff
@@ -180,82 +180,65 @@ def __init__(self, file_path):
         self.image_masks = None
         self._roi_locs = None
         self._accepted_list = None
+        self._rejected_list = None
         self._io = NWBHDF5IO(file_path, mode='r+')
```

Comment:
why r+ instead of r?

Reply:
changed https://github.com/catalystneuro/roiextractors/pull/46/commits/0a83bb80369f51297c80456895dc94c5c426b101